components/screens/CalendarScreen/CalendarScreen.js

```diff
@@ -1,11 +1,54 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, {useState, useEffect} from 'react';
+import { View, Text, ScrollView} from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { Styles } from './CalendarStyle';
+
+//fetch data from the server
+const calendarURL = 'https://cascade-api.herokuapp.com/calendar';
 
 const CalendarScreen = () => {
+
+   const [isLoading, setLoading] = useState(true);
+   const [calendarData, setCalendarData] = useState([]);
+
+
+
+  useEffect(() => {
+    fetch(calendarURL)
+      .then(response => response.json())
+      .then(json =>
+         setCalendarData(json))
+      .catch(error => alert(error))
+      .finally(setLoading(false));
+  }, []);
+
+  //console.log(calendarData);
+
+  //rendering
   return (
-    <View>
-      <Text>Calendar View</Text>
-    </View>
+    <SafeAreaView>
+    <ScrollView>
+     
+      {calendarData.map(({title, url, date, pace, distance, leader }) => (
+        <>
```

Comment:
This empty container should be changed to a `<View>` with a unique key given. This will remove some of the console warnings. _See Alex's current [PR ](https://github.com/brendenwest/cascade/pull/43) for an example of this._

Reply:
Noted and applied.